composer.json

```diff
@@ -12,7 +12,11 @@
     "require": {
         "php": "~7.0",
         "phpstan/phpstan": "^0.10",
-        "typo3/minimal": "^8.7.0 || ~9.0"
+        "typo3/cms-core": "^8.7.0 || ~9.0",
```

Comment:
`^9.5` would mean only LTS (9.5.*) and `~9.0` would mean beginning from 9.0 to 9.x. So it would also support 9.4... And there is no need to exclude `> 9.0 && < 9.5`, right? Nobody should use a version 9 lower than 9.5 for sure...

Reply:
Exactly. And since it doesn't make sense to use any 9.x besides 9.5 there is no point supporting it. :-)